db/db_test.cc

```diff
@@ -2743,7 +2743,10 @@ TEST_P(MultiThreadedDBTest, MultiThreaded) {
     thread[id].state = &mt;
     thread[id].id = id;
     thread[id].multiget_batched = multiget_batched_;
-    env_->StartThread(MTThreadBody, &thread[id]);
+    // Cannot use env_->StartThread because the thread needs to be detached.
```

Comment:
This doesn't make sense. env_->StartThread *only* creates detached threads. If it's not working, it needs to be fixed.

And this code (probably ancient) would be much better off with joinable threads anyway.

Reply:
Already changed the code from detached threads to joinable threads